modules/oe_content_publication/config/post_updates/00003_create_fields/field.field.node.oe_publication.oe_publication_type.yml

```diff
@@ -0,0 +1,35 @@
+langcode: en
```

Comment:
This field already exist - no need to create it.

Reply:
I removed the file.